api/app/src/main/resources/application-test.properties

```
@@ -1,7 +1,8 @@
-spring.datasource.driver-class-name=org.h2.Driver
-spring.datasource.url=jdbc:h2:mem:test
-spring.datasource.username=sa
-spring.datasource.password=
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
+spring.datasource.url=jdbc:postgresql://localhost:5432/packit?stringtype=unspecified
+spring.datasource.username=packituser
+spring.datasource.password=changeme
 spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```

```suggestion
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL82Dialect
```
Recommended to have this in test config
`spring.jpa.hibernate.ddl-auto=create`

---

Because we're no longer using an embedded db but using a db that is already initialised (the dockerised Postgres db) we don't want Hibernate to try and initialise it.